workflow.txt

```diff
@@ -1,10 +1,10 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
 git add .
+git commit -m ""
 git pull upstream master
+git push origin my-solution
```

Comment:
👍 Going to mark as correct, but question was asking for the normal commands to get to hw submission, i.e:
```
fork on github
git clone
git checkout -b my-solution
git add .
git commit -m ""
git push origin my-solution
```

that would get you to where you could make a pull request on github